source/hardware/controllers/denon_mc7000.rst

```diff
@@ -1,18 +1,12 @@
 Denon MC7000
 ============
 
-The Denon MC7000 is a professional DJ controller which has got 4-channel
-capability and dual USB connections. These two USB audio interfaces
-enable two DJs to play together. This controller includes Denon’s high
-build quality and superior 24-bit audio reproduction, makes this suited
-to both mobile and club DJs.
-
-The dual 6" platters have a touch-capacitive design with rotational LED
-displays. The MC7000 has exclusive and dedicated Key Matching and
-Changing controls that deliver an unprecedented ability to instantly and
-harmonically match musical keys. The unit’s comprehensive, multi-input
-mixer also provides access to line/phono inputs from both authentic
-analogue (vinyl) and digital sources.
+The Denon MC7000 is a professional DJ controller which has got 4-channel capability and dual USB connections. These two USB audio interfaces enable two DJs to play together. This controller includes Denon’s high build quality and superior 24-bit audio reproduction, makes this suited to both mobile and club DJs.
+
+The dual 6" platters have a touch-capacitive design with rotational LED displays. The MC7000 has exclusive and dedicated Key Matching and Changing controls that deliver an unprecedented ability to instantly and harmonically match musical keys. The unit’s comprehensive, multi-input mixer also provides access to line/phono inputs from both authentic analogue (vinyl) and digital sources.
+
+Links
+~~~~~
```

Comment:
I don't think we have a separate "links" headline in the other controller docs.

```suggestion
```

Reply:
easy to remove it if you don't like.